src/lib/components/molecules/fptp-waffle/index.jsx

```diff
@@ -0,0 +1,14 @@
+import { Waffle, HalfLineText } from '$particles'
+import defaultStyles from "./style.module.css"
+import { mergeStyles } from "$styles/helpers/mergeStyles"
+
+export const FPTPWaffle = ({ total, units, rows, abbreviationAccessor, halfLineTextHeight, paddingTop, onMouseOver, squares, styles }) => {
+  styles = mergeStyles(defaultStyles, styles)
+  
+  return (
+    <div class={styles.wrapper}>
+      <HalfLineText text={`${Math.ceil(total / 2)} for a majority`} height={halfLineTextHeight} />
```

Comment:
This could be a simple `<p>` tag, and I think the height doesn't need to be hardcoded.